src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/Configuration/IProjectConfigurationDimensionsProvider4.cs

```diff
@@ -0,0 +1,12 @@
+﻿using System.Collections.Generic;
+using System.Collections.Immutable;
+
+using Microsoft.Build.Construction;
+
+namespace Microsoft.VisualStudio.ProjectSystem
+{
+    internal interface IProjectConfigurationDimensionsProvider4 : IProjectConfigurationDimensionsProvider3
```

Comment:
Since its only used by us, should this name be something different, lest CPS decide they need a version 4 of the interface?

Reply:
This is my lazy ass not putting this into CPS where it belongs. I'll file a bug.